src/features/campaigns/components/ActivityList/items/EmailListItem.tsx

```diff
@@ -22,40 +22,40 @@ const statusColors = {
 const EmailListItem: FC<EmailListItemProps> = ({ orgId, emailId }) => {
   const { data: email } = useEmail(orgId, emailId);
   const state = useEmailState(orgId, emailId);
-  const { data: emailStats } = useEmailStats(orgId, emailId);
+  const {
+    data: emailStats,
```

Comment:
Couldn't you get the separate properties now returned from `useEmailStats()` instead of `data`?

Reply:
I agree, i thought about it like one minute after i made the pr heh. i'll fix it :D